python/GafferCortexUI/OpDialogue.py

```diff
@@ -500,3 +504,22 @@ def __setUserDefaults( self, graphComponent ) :
 
 		for child in graphComponent.children() :
 			self.__setUserDefaults( child )
+
+class _OpDialogueEventFilter( QtCore.QObject ) :
+
+	def __init__( self ) :
+
+		QtCore.QObject.__init__( self )
+
+	def eventFilter( self, qObject, qEvent ) :
+		type = qEvent.type()
+
+		if type == QtCore.QEvent.Show :
+			widget = GafferUI.Widget._owner( qObject )
+			widget.resizeToFitChild()
+			return True
+
+		return False
+
+# this single instance is used by all op dialogue widgets
+_opDialogueEventFilter = _OpDialogueEventFilter()
```

Comment:
If this were in one of the official Gaffer modules, I'd want to exhaust all simpler approaches before resorting to an event filter. But I think it's time GafferCortex was hosted at IE, in which case it doesn't matter what I think, so I'm going to use this PR to broach the topic with Ivan.